setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name="tap-frontapp",
-    version="1.1.0",
+    version="1.1.1",
```

Comment:
```suggestion
    version="1.1.0",
```

Reply:
version change not necessary for circle-ci config

Reply:
releasing it along with #18 hence, added